extension/data/modules/macros.js

```diff
@@ -441,6 +447,12 @@ export default new Module({
                         $body.find('.ThreadViewer .InfoBar__control:not(.m-on) .icon-mute').click();
                     }
 
+                    if (userflair) {
+                        TBApi.flairUser(info.author, info.subreddit, null, null, userflair).catch(() => {
+                            status.text(`error, failed to flair user (${userflair})`);
```

Comment:
`status.text` doesn't exist. It looks like most of our error reporting in this file is done via `TBui.textFeedback('...', TBui.FEEDBACK_NEGATIVE)`, so we might as well use that:

```suggestion
                            TBui.textFeedback(`error, failed to flair user (${userflair})`, TBui.FEEDBACK_NEGATIVE);
```

Error reporting for this module needs to be redone at some point to support reporting multiple errors at the same time anyway, but this will be good enough for now I think.

Reply:
Yup you're right, updated the PR with suggested change.